deployment/entity-service/values.yaml

```diff
@@ -9,14 +9,24 @@ anonlink:
   config: {}
 
   objectstore:
-    uploadEnabled: "true"
-    # TODO can't leave defaults once minio exposed
+    ## Disable the feature providing client's with restricted upload access to the object store.
+    ## By default we don't expose the Minio object store, however it needs to be exposed when
+    ## enabling this feature. See section `minio.ingress`.
```

Comment:
I don't understand this comment. So what do I have to set this to to enable client upload to our Minio object store?

Reply:
I'll clarify this